python/numpydoc.rst

```diff
@@ -73,14 +73,21 @@ Docstrings SHOULD begin with ``"""`` and terminate with ``"""`` on its own line
 
 The docstring's :ref:`summary sentence <py-docstring-short-summary>` occurs on the same line as the opening ``"""``.
 
-The terminating ``"""`` should be on its own line, even for 'one-line' docstrings (this is a minor departure from :pep:`257`).
-For example, a one-line docstring:
+The terminating ``"""`` should be on its own line except for one-line docstrings.
+If the docstring is a single line, the terminating ``"""`` may be either on the same line or on its own line.
+(Be aware that :pep:`257` requires that it be on the same line and black will enforce this rule.)
```

Comment:
In `python/style.rst`, double-backticks are used for "black".  Actually, I wonder if it should be a link in both places...